meetings/2022-11-17.md

```diff
@@ -0,0 +1,128 @@
+
+
+### Simplify flow diagrams
+URL: https://github.com/solid/notifications/pull/108
+
+* SC: Any objections to merge?
+* eP: It is still approved from me and has requested changes from Sarven
+* SC: I see it is the loop part. No objections to merge it with it in.
+* SC: Rahul are you ok with this?
+* RG: +1 I'm good with it.
+* SC: Merging
+
+
+### Considerations
+
+* SC: We may be able to process these security considerations quickly.
+* RG: You need to explain what is the context here.
+
+
+#### Add consider-exposing-information
+URL: https://github.com/solid/notifications/pull/117
+
+* SC: Exposing information advises to avoid exposing any information that is not required for the interaction. For example when notification is provided, it may include which resource was changed but not which agent changed it. We don't want servers to just slip in such information.
+* RG: Do you plan to include it as normative? I think that for the server it should maybe be required.
+* SC: We don't want to say that something is non-conforming just because something added some information.
+* ...: When someone is writing implementation they can use it as a reference when they review it.
+* ...: In some environments some constraints may be provided, and something would needed to be included in the payload. We don't want to prohibit such use scenarios.
+* eP: Makes sense to me. I also don't think we can make it normative.
+* RG: I need to think about it and see if we need to add something normative for the server.
+* SC: We don't want to limit in what ways the protocol can be used. We could mandate closed list of information but that would be very limiting.
+* RG: For the client I'm not worrying about what it sends. For the server I see three buckets, what must be send and whay may be send. Te former we can specify what must not be send.
+* SC: I don't think we can specify what must not be send.
+* eP: Do you have some concrete example?
+* RG: For example exposing access control.
+* SC: It is possible that administrator or owner may want to track changes to access control.
+* RG: In the document I created I made a list of what would be the minimum information that has to be send. We could take a look at it.
+
+#### Add consider-subscription-resource-trust
+URL: https://github.com/solid/notifications/pull/118
+
+* SC: Client should be careful where it makes request. Especially a localhost, if I know that you have a server you your localhost, I could made you make request to your local server repeatedly.
+* RG: How do you stop this on the web. I can point to subscription on your server and clients would go make subscriptions there. Now that server would start getting subscription requests.
+* SC: There are legitimate reasons to have something on localhost. We just want to prevent clients blindly making requests. For example that client can have allow and deny lists of origins or particular resources. 
```

Comment:
```suggestion
* SC: There are legitimate reasons to have something on localhost. We just want to prevent clients blindly making requests. For example, that client can have allow and deny lists of origins or particular resources. 
```